Adding viewBounds function to return map view bounds of all corners i…

jjozwiak opened the pull request:
…n Lat/Lng. Based on contribution by @J-Gonzalez PR #81 but updated to match project structure now being used in main branch

selected-pixel-jameson commented:
@tafelnl If we get the conflicts resolved on this can we get this merged?

tafelnl commented on src/web.ts:
```diff
@@ -76,6 +77,10 @@ export class CapacitorGoogleMapsWeb
     throw this.unimplemented("Not implemented on web.");
   }
 
+  async viewBounds(_options: ViewBoundsOptions): Promise<any> {
```
Let's rename this to `getViewBound` to be consistent with the rest of the API.

tafelnl commented on src/definitions.ts:
```diff
@@ -67,6 +68,8 @@ export interface CapacitorGoogleMapsPlugin {
 
   removeMarker(options: RemoveMarkerOptions): Promise<void>;
 
+  viewBounds(options: ViewBoundsOptions): Promise<any>;
```
We should add a return type to keep TypeScript working.

tafelnl replied:
We should add a return type to keep TypeScript working.

tafelnl reviewed (changes requested):
Thanks for your PR. Just reviewed your it. Can you take a look and also resolve the merge conflicts?

jjozwiak commented:
Hi @tafelnl , thanks for the review and feedback. I rebased this branch, resolved merge conflicts, and implemented your feedback. Please let me know if there's anything else. Cheers

jjozwiak commented:
Hi @tafelnl,
I pushed the requested updates and signed the commits for this PR. Is there anything else needed to merge this into main?
Thank you

selected-pixel-jameson commented:
Hi @tafelnl. Is there anyway we can get this merged? Trying to take a project to production and would like to use this repo instead of our fork if at all possible. Thank you!

selected-pixel-jameson commented:
Hi @tafelnl. Is there anyway we can get this merged? Trying to take a project to production and would like to use this repo instead of our fork if at all possible. Thank you!

selected-pixel-jameson commented:
@tafelnl Is there something we can do to get this merged? Are we missing something?

igor-xx commented:
@tafelnl @hemangsk  could anoyone please merge this? or is there anything else preventig you from doing it?

Jador-de-Vries commented:
Bumping as I'd also like to see this implemented!
